spokestack/extensions/webrtc/vad.pyx

```diff
@@ -0,0 +1,41 @@
+cimport cvad
+cimport numpy as np
+
+
+np.import_array()
+
+cdef class WebRtcVad:
+    cdef cvad.VadInst*_vad
+    cdef int _sample_rate
```

Comment:
the `_sample_rate` and `_mode` members are only used in the constructor, so they don't need to be stored